en/Licenses & Payment/Commercial license.md

```diff
@@ -51,17 +51,20 @@ For bulk discounts, contact support@obsidian.md.
 > [!FAQ]- Q9: I'm a freelancer/writer/blogger, do I need a commercial license?
 > Unless you hire at least one additional person, you do not need a commercial license.
 
-> [!FAQ]- Q10: I'm a teacher/professor and I use Obsidian for my teaching/lectures, do I need a commercial license?
+> [!FAQ]- Q10: I'm a teacher/professor or I work at a school and I use Obsidian for my teaching/lectures, do I need a commercial license?
 > Obsidian usage for education-related activities within schools and other recognized educational institutions does not require a commercial license. 
 
 > [!FAQ]- Q11: I'm an academic researcher and I use Obsidian for my research work. Do I need a commercial license?
 > If your research involves producing a commercial product (e.g., if your research contributes to revenue-generating commercial activities), you need a commercial license for your usage. If your research does not contribute to a commercial product, or if your research has not yet led to commercial activities, you do not need a license.
 
-> [!FAQ]- Q12: I use Obsidian during work for things like writing down team processes and taking notes for team meetings, do I need a commercial license?
+> [!FAQ]- Q12: I work for a Government or an arms-length governmental agency. Do I need a commercial license?
+> Yes, you need a commercial license. From our EULA, if you use OBSIDIAN for commercial use, you must obtain a commercial license. Commercial use is defined as using OBSIDIAN for work-related activities in a company with two (2) or more employees. (The EULA's revenue-generating clause exempts non-profit organizations.)
```

Comment:
```suggestion
> Yes, you need a commercial license. From our EULA, if you use OBSIDIAN for commercial use, you must obtain a commercial license. Commercial use is defined as using OBSIDIAN for work-related activities in a company with two (2) or more employees. (The EULA's revenue-generating clause exempts non-profit organizations.)
```

This looks like a quote from the EULA. I think we should considering putting it in blockquotes to let users know it's the actual text rather than a summary.

Reply:
@marcusolsson The only quoted text is "work-related activities in a company with two (2) or more employees" — up to you whether you'd prefer to block-quote that!

Not sure how Obsidian got casplock'd. I'd write it normally.

Reply:
```suggestion
> Yes, you need a commercial license. From our EULA, if you use Obsidian for commercial use, you must obtain a commercial license. Commercial use is defined as using Obsidian for "work-related activities in a company with two (2) or more employees". The EULA's revenue-generating clause exempts non-profit organizations.
```

How about surrounding the actual excerpt with quotes then? 

Also, I don't think the parentheses are necessary here.

Reply:
Looks great to me!